CMakeLists.txt

```
@@ -111,6 +111,18 @@ if (USE_CCACHE)
     endif()
 endif()
 
+#################################
+# Scafacos Package (optional)   #
+#################################
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(SCAFACOS IMPORTED_TARGET scafacos)
```

```suggestion
find_package(PkgConfig)
if(PkgConfig_FOUND)
  pkg_check_modules(SCAFACOS IMPORTED_TARGET scafacos)
endif()
```
Let's not required `pkg-config` for an optional package.

---

Updated